week-3/A-array-find/exercise.js

```diff
@@ -4,8 +4,18 @@
   Using .find() (and any other array methods necessary), find the lowest pass score
 */
 
+function findLowestPassScore(scores) {
```

Comment:
`findLowestPassScore` isn't a good name for this function, given what it does - it takes a single score and returns `true` if it is a pass and `false` otherwise. What would be a better name. The parameter name is also confusing, since it is plural.

Reply:
Yeah, i reckon  "isPass " would be a better name for the function. 
I have also changed the parameter name to singular , "score".
I will push the changes soon .